synthtool/languages/java.py

```diff
@@ -94,6 +95,7 @@ def fix_proto_headers(proto_root: Path) -> None:
         f"{GOOD_LICENSE}{PROTOBUF_HEADER}",
     )
     s.replace([proto_root / "src/**/*Name.java"], BAD_LICENSE, GOOD_LICENSE)
```

Comment:
You can pass a list to replace
```suggestion
     s.replace([proto_root / "src/**/*Name.java", proto_root / "src/**/*Names.java"], BAD_LICENSE, GOOD_LICENSE)
```

Reply:
Done.